Jenkinsfile

```diff
@@ -111,6 +111,10 @@ pipeline {
                 environment name: "DONT_CREATE_TESTBEDS", value: null
                 expression { return !currentBuild.result }
             }
+            environment {
+                TESTBED_TEMPLATE = "Template-testbed-201802130923"
```

Comment:
This is ok for this PR, but I will add to backlog that it should have a hash instead of timestamp.